bin/reth/src/node/mod.rs

```
@@ -88,6 +89,9 @@ impl Command {
         let config: Config = confy::load_path(&self.config).unwrap_or_default();
         info!("reth {} starting", crate_version!());
 
+        let new_limit = raise_fd_limit().expect("Failed to raise the fd limit of the binary");
```

this should return an error instead, see `.wrap_err`